packages/dialtone/lib/build/less/components/chip.less

```diff
@@ -128,6 +128,12 @@
   background-color: var(--dt-color-black-400);
 }
 
+.d-chip__text {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+}
```

Comment:
Where this come from? From `d-truncate`?

Reply:
Hmm , `d-truncate` = 
` text-overflow: ellipsis !important;
    white-space-collapse: collapse !important;
    text-wrap: nowrap !important;
    overflow: hidden !important;`

Seems like we left `text-wrap: nowrap`

Reply:
this is ok